G/GiNaC/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "GiNaC"
+version = v"1.8.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.ginac.de/ginac-$(version).tar.bz2", "00b320b1116cae5b7b43364dbffb7912471d171f484d82764605d715858d975b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ginac-*.*.*/
+./configure --prefix=$prefix --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(; exclude=Sys.iswindows) # Because the `Python_jll` is not built for Windows platforms now.
+platforms = expand_cxxstring_abis(platforms)
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libginac", :libginac),
+    ExecutableProduct("viewgar", :viewgar),
+    ExecutableProduct("ginsh", :ginsh)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="Bison_jll", uuid="0f48145f-aea8-549d-8864-7f251ac1e6d0"))
+    Dependency(PackageSpec(name="CLN_jll", uuid="b3974076-79ef-58d3-b5c7-5ef926e97925"))
+    Dependency(PackageSpec(name="flex_jll", uuid="48a596b8-cc7a-5e48-b182-65f75e8595d0"))
```

Comment:
Are you sure flex is a runtime dependency?

Reply:
From the logs, it seems that `configure: WARNING: yywrap not found; giving up on flex` forbids the flex.
I'm checking what was wrong.

Reply:
Actually, I found the `yywrap` in the `libfl.so`:
```shell
readelf -s ${WORKSPACE}/destdir/lib/libfl.so
```
with
```log
...
    54: 0000000000000550     1 FUNC    GLOBAL DEFAULT    6 _init
    55: 0000000000000740     6 FUNC    GLOBAL DEFAULT    8 yywrap
    56: 0000000000000000     0 NOTYPE  WEAK   DEFAULT  UND __deregister_frame_info
...
```